spec/requests/organization_requests_spec.rb

```diff
@@ -184,6 +184,22 @@
       sign_in(@super_admin)
     end
 
+    describe "GET #show" do
+      before { get organization_path(default_params) }
+
+      it { expect(response).to be_successful }
+
+      it 'organization details' do
+        expect(assigns(:organization)).to have_attributes(
```

Comment:
I don't see this spec as super helpful - you're diving way deep into the weeds of implementation. I'd rather look at the rendered text and verify that the value you expect to see is there. In this case you should use `travel_to` to lock in the date and time, update the organization to have specified values, and then expect those values to be in the output.

Reply:
Ok I will take a look at this again. Sorry, I have not forgotten about this. I just got a new Macbook and am still in the process of getting things up and running!

Reply:
@dorner Starting to take a look at this now and trying to wrap my head around your suggestion. Sorry, I'm still relatively newer and still learning a lot. Do you know if there is something similar or an example I can look at?

Reply:
Here's an example of the traveling: [https://github.com/rubyforgood/human-essentials/blob/f3379f30e42a05259e717d3f5bc5444cf3977ad8/spec/models/distribution_spec.rb#L113-L113](https://github.com/rubyforgood/human-essentials/blob/f3379f30e42a05259e717d3f5bc5444cf3977ad8/spec/models/distribution_spec.rb#L113-L113)

and here's how you can check the rendered text: 

[https://github.com/rubyforgood/human-essentials/blob/f3379f30e42a05259e717d3f5bc5444cf3977ad8/spec/requests/admin_requests_spec.rb#L22-L22](https://github.com/rubyforgood/human-essentials/blob/f3379f30e42a05259e717d3f5bc5444cf3977ad8/spec/requests/admin_requests_spec.rb#L22-L22)

You can get more nitty-gritty by actually turning the text into DOM and checking it specifically: https://github.com/rubyforgood/human-essentials/blob/6121c0ee52cf562c1b9fee82e0e6345d5824aef7/spec/requests/kit_requests_spec.rb#L25-L25

Reply:
I'm not sure we actually need to travel-to   to test this out -- we're looking at the issued_at, rather than the created_at for the last  distribution date, and that is "manually"  set.   Or am I missing something?

Reply:
My bad - as usual @cielf is right :smiley:

Reply:
So can I resolve this conversation and we're good to go with this change?

Reply:
No - it's still an issue IMO. You shouldn't be looking at `assigns` - it's better to look at the rendered text.

Reply:
This took _waaaay_ longer than I had hoped and expected. I think I figured it out though and, of course, it seems to be way simpler than I had made it out to be. 
Sorry, that it took so long.... still learning over here :upside_down_face: